Apps/WebClient/src/ClientApp/src/views/faq.vue

```diff
@@ -0,0 +1,7 @@
+<template>
```

Comment:
@FuriousLlama can we remove this to avoid the conflicts with my PR? As I believe you needed this only for your local testing right?

Reply:
should be an easy merge, since the file names are the same. But I can remove it

Reply:
sounds good, I can approve it then.